page-home.php

```diff
@@ -0,0 +1,123 @@
+
+<?php
+/*
+ * Template Name: Currency Converter
+ *
+ */
+get_header(); ?>
+
+<?php get_template_part( 'breadcrumb' ); ?>
+
+<div id="primary" class="content-area level-one">
+    <div class="container">
+        <div class="row" role="banner">
+            <div class="col-md-12">
+                <?php
+                global $post;
+                $image      = make_path_relative_no_pre_path( get_feature_image_url( $post->ID, 'full-page-width', true ) );
+                $title      = get_the_title();
+                get_page_banner( 'level one', $title, $image, '', '', '' );
+                ?>
+            </div>
+        </div>
+
+            <div class="row">
+                <main role="main">
```

Comment:
remove `role="main"` as it's not needed. The tag is self explanatory.

Reply:
Hi @punalsc , I was advised by @gtvj to keep this as no all browsers support <main but do support role="main"

Reply:
Hi @punalsc @AshTNA - I did say this. The `<main>` tag is not supported in all the browsers we support so they won't be reporting the semantic information via the accessibility API. `<main role="main">` seems redundant but the role provides a mechanism to report where ARIA is available.

Reply:
Hi @AshTNA, @gtvj I have just checked the browser compatibility on MDN https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#Browser_compatibility and yes your right, it clearly suggests for IE 11 and lower, the `role=main` should be implemented. I'l mark this as resolved.